pyvista/plotting/lights.py

```diff
@@ -142,9 +221,36 @@ def __eq__(self, other):
 
     def __del__(self):
         """Clean up when the light is being destroyed."""
-        self._actor = None
+        self.actor = None
+        self._renderers = []
```

Comment:
Mild preference to use the original list, should there be a stray reference to the list of renderers elsewhere.
```suggestion
        self._renderers.clear()
```
(cf. `Plotter.clear`)

Reply:
Good point.